engines/director/castmember.cpp

```diff
@@ -1296,14 +1296,20 @@ void TextCastMember::updateFromWidget(Graphics::MacWidget *widget) {
 }
 
 Common::String TextCastMember::formatInfo() {
+	Common::String readable = _ptext.encode();
+	for (int i = 0; i < (int)readable.size(); i++) {
+		if (readable[i] == '\r')
```

Comment:
What about \r\n?

But in general, I'd recommend using `String::toPrintable()`

Reply:
Changed the Datum printable string formatter and used that. There is a preprocess step to turn "\r" and "\r\n" into "\n" so that Mac multiline strings display properly.